stats/HTTP.md

```diff
@@ -88,13 +87,23 @@ starts and are available in the context for the entire incoming request processi
 `http_server_status` is set when an incoming request finishes and is only available around the stats
 recorded at the end of request processing.
 
+`http_server_path` and `http_server_host` are set by the client: you should be careful about using these
```

Comment:
Maybe same comment and changes should be added to the client side stats.

Reply:
Removed path from HTTP client views as well